upgrader/apps/upgrader/lib/utility/UpgradeUtility.php

```diff
@@ -382,10 +382,22 @@ public function getNotes($startIncNumber, $endIncNumber) {
     }
 
     public function getInstalledAddons() {
+        if (!$this->tableExists('ohrm_marketplace_addon')) {
+            return [];
+        }
         $query = 'select * from ohrm_marketplace_addon where plugin_name is not null';
         return mysqli_fetch_all($this->executeSql($query), MYSQLI_ASSOC);
     }
 
+    public function tableExists($tableName) {
+        $sql = "SHOW TABLES LIKE '$tableName'";
```

Comment:
can't use as a name param ?
 $sql = "SHOW TABLES LIKE %:tableName%";

Reply:
I don't think it's really necessary as this function is never going to invoked with user inputs.